lib/chrome-driver.js

```diff
@@ -24,7 +23,7 @@ ChromeDriver.prototype.start = function () {
     cwd: process.cwd(),
     env: this.getEnvironment()
   }
-  this.process = ChildProcess.spawn(process.execPath, args, options)
+  this.process = ChildProcess.spawn(this.path, args, options)
```

Comment:
Will this work on Windows? I think I originally used `process.execPath` to get it working on Windows.